app/views/curation_concerns/file_sets/upload/_script_templates.html.erb

```diff
@@ -1,8 +1,7 @@
-<!-- TODO: make the upload_set_id available on the presenter -->
-<% if false && presenter.upload_set_id %>
+<% if @presenter && @presenter.upload_set_id %>
```

Comment:
@hectorcorrea yes, but that controller has set `@presenter`, right?  If it doesn't we want it to raise an error.  Every time this page is rendered we want to require a `@presenter`